partitioned-table.md

```diff
@@ -366,20 +366,20 @@ After creating the partitions as above, you can easily add or delete records rel
 
 You can also execute `ALTER TABLE employees DROP PARTITION pEast` to delete all related rows, but this statement also deletes the `pEast` partition from the table definition. In this situation, you must execute the `ALTER TABLE ... ADD PARTITION` statement to recover the original partitioning scheme of the table.
 
-Unlike Range partitioning, List partitioning does not have a similar `MAXVALUE` partition to store all values that do not belong to other partitions. Instead, all expected values of the partition expression must be included in the `PARTITION ... VALUES IN (...)` clause. If the value to be inserted in an `INSERT` statement does not match the column value set of any partition, the statement fails to execute and an error is reported. See the following example:
+If the new feature Default List Partition, controlled by `tidb_enable_default_list_partition` variable is not used, then unlike Range partitioning, List partitioning does not have a similar `MAXVALUE` partition to store all values that do not belong to other partitions. Instead, all expected values of the partition expression must be included in the `PARTITION ... VALUES IN (...)` clause. If the value to be inserted in an `INSERT` statement does not match the column value set of any partition, the statement fails to execute and an error is reported. See the following example:
```

Comment:
@mjonss  Would you please confirm whether tidb_enable_default_list_partition is a new system variable? If yes, would you please add its description to system-variables.md in this PR?

Reply:
Hi, it is a new system variable. I will add it to system-variables.md